Update BUILDING.md

## Background

I am new to Android Studio. When I was following the instructions in `BUILDING.md`, although I was able to get the app running eventually, I ran into some issues:

**1. Running `npm start` gives the following error:**
```
npm ERR! Missing script: "start"
npm ERR! 
npm ERR! Did you mean one of these?
npm ERR!     npm star # Mark your favorite packages
npm ERR!     npm stars # View packages marked as favorites
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run
```
Steps to reproduce: 
1. cd into `peri` folder
2. run `npm install`
3. run `npm start`

Question: Is there a different way to start the development server?

**2. and build the project based on this instruction https://capacitorjs.com/docs/android**

When I followed the instructions in "build the project based on this instruction https://capacitorjs.com/docs/android", following that page's section titled **Running on the Command-Line** resulted in a build failure

Suggested fix: Update `BUILDING.md` to instruct users to follow the instructions in **Running with Android Studio**. That worked for me. 

If these issues are happening for others, I can update the instructions in the README, as it would help new contributors.

--- a/BUILDING.md
+++ b/BUILDING.md
@@ -9,8 +9,10 @@ npm install
 After that, you can start development server, which will be useful for quickly testing your changes:
 
 ```shell
-npm start
+npm run dev
 ```
+
+Once your dev server is running, you should see a link like `http://localhost:5173/` in your shell. Copy the link and paste it into your browser to see the running app.
 
 To build the native `peri` you need `Android Studio` installed and all necessary JDK/SDK. To install, follow the instruction <https://developer.android.com/studio/install>.
 
@@ -24,7 +26,7 @@ npx cap add android
 npx cap open android
 ```
 
-An `android` directory will be generated and you can open it with `Android Studio` and build the project based on this instruction <https://capacitorjs.com/docs/android>
+An `android` directory will be generated and you can open it with `Android Studio` and build the project with the instructions in <https://capacitorjs.com/docs/android#running-with-android-studio>, starting from the **Running with Android Studio** section.
 
 You can also create a development version of native `peri` using the following `Dockerfile`, just create it in the root of the `peri` project:
 

